packages/ui/src/views/repo/repo-commits/repo-commits-view.tsx

```diff
@@ -41,11 +42,13 @@ export const RepoCommitsView = (props: RepoCommitsViewProps) => {
         </Text>
         <Spacer size={6} />
         <div className="flex justify-between gap-5">
-          {!props.isFetchingBranches && branchList && (
+          {branchList && (
```

Comment:
not a blocker, but I think we should always show the branch selector, but disable it with a loader until branch list is available

Reply:
agreed, will fix this

Reply:
branchList can change whilst we are typing in the searchbox as it is now hooked to the api based on the query, so this approach would break the component.

Reply:
will remove the check for branchList as that is handled within nodata state inside the BranchSelector